src/Listener/ApiListener.php

```diff
@@ -149,7 +149,7 @@ protected function _checkRequestMethods()
             }
         }
 
-        throw new BadRequestException('Wrong request method');
+        throw new MethodNotAllowedException('Wrong request method');
```

Comment:
The message string can also be removed and instead just let the default message for this exception be used.